src/main/java/com/opsmatters/newrelic/api/services/GraphQLService.java

```diff
@@ -114,6 +126,18 @@ private String constructNrqlQuery(int accountId, String query) {
                 "}";
     }
 
+    private String constructNrqlQueryWithTimeout(int accountId, String query, int timeout) {
+        return "{" +
+                "  actor {" +
+                "    account(id: " + accountId + ") {" +
+                "      nrql(query: \"" + query + "\", timeout: " + timeout + ") {" +
```

Comment:
A lot of graphQL APIs use it as an API that calls other APIs, so it may be setting an internal timeout. We usually want to control this from the OkHttp Client in the backend, but if the default is 5 seconds I am guessing this is an internal timeout since ours is 30 seconds